@gtmanfred Looks like your new test needs a little ❤️ 

Can you take a look?

- unit.modules.test_boto_route53.BotoRoute53TestCase.test_create_healthcheck
```
Error Message

'CreateHealthCheckResponse'
Stacktrace

Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/moto/core/models.py", line 71, in wrapper
    result = func(*args, **kwargs)
  File "/testing/tests/unit/modules/test_boto_route53.py", line 124, in test_create_healthcheck
    del healthcheck['CreateHealthCheckResponse']['HealthCheck']['CallerReference']
KeyError: 'CreateHealthCheckResponse'
```

--- a/tests/unit/modules/test_boto_route53.py
+++ b/tests/unit/modules/test_boto_route53.py
@@ -99,18 +99,20 @@ class BotoRoute53TestCase(TestCase, LoaderModuleMockMixin):
         register_instances returns True.
         '''
         expected = {
-            'CreateHealthCheckResponse': {
-                'HealthCheck': {
-                    'HealthCheckConfig': {
-                        'FailureThreshold': '3',
-                        'IPAddress': '10.0.0.1',
-                        'ResourcePath': '/',
-                        'RequestInterval': '30',
-                        'Type': 'HTTPS',
-                        'Port': '443',
-                        'FullyQualifiedDomainName': 'blog.saltstack.furniture',
+            'result': {
+                'CreateHealthCheckResponse': {
+                    'HealthCheck': {
+                        'HealthCheckConfig': {
+                            'FailureThreshold': '3',
+                            'IPAddress': '10.0.0.1',
+                            'ResourcePath': '/',
+                            'RequestInterval': '30',
+                            'Type': 'HTTPS',
+                            'Port': '443',
+                            'FullyQualifiedDomainName': 'blog.saltstack.furniture',
+                        },
+                        'HealthCheckVersion': '1',
                     },
-                    'HealthCheckVersion': '1',
                 },
             },
         }
@@ -121,6 +123,6 @@ class BotoRoute53TestCase(TestCase, LoaderModuleMockMixin):
             port=443,
             resource_path='/',
         )
-        del healthcheck['CreateHealthCheckResponse']['HealthCheck']['CallerReference']
-        del healthcheck['CreateHealthCheckResponse']['HealthCheck']['Id']
+        del healthcheck['result']['CreateHealthCheckResponse']['HealthCheck']['CallerReference']
+        del healthcheck['result']['CreateHealthCheckResponse']['HealthCheck']['Id']
         self.assertEqual(healthcheck, expected)

